crates/orderbook/src/database/total_surplus.rs

```diff
@@ -0,0 +1,72 @@
+use {
+    anyhow::Result,
+    database::{byte_array::ByteArray, Address},
+    primitive_types::{H160, U256},
+    sqlx::PgConnection,
+};
+
+/// Computes a user's total surplus received (price improvement over limit price
+/// and **NOT** quoted price) since march 2023.
+async fn fetch_total_surplus(ex: &mut PgConnection, user: &Address) -> Result<f64, sqlx::Error> {
+    const TOTAL_SURPLUS_QUERY: &str = r#"
+with trade_components as (
+    select
```

Comment:
Could you make the SQL keywords all caps? Just to be consistent with how queries are usually written both in our code and elsewhere?